bucket/cormanlisp.json

```
@@ -0,0 +1,42 @@
+{
+    "version": "3.1.2b",
+    "description": "Corman Lisp development environment for Common Lisp",
```

Do not mention application name. It is obsolete

```suggestion
    "description": "Common Lisp development environment",
```